src/Components/Search/Previews/Grids/ArtistSearch/MarketingCollections.tsx

```diff
@@ -1,10 +1,9 @@
-import { data as sd } from "sharify"
-
 import { Box, color, Flex, Link, Sans, Serif, space } from "@artsy/palette"
 import { MarketingCollectionsPreview_marketingCollections } from "__generated__/MarketingCollectionsPreview_marketingCollections.graphql"
 import React from "react"
 import { createFragmentContainer, graphql } from "react-relay"
 import styled from "styled-components"
+import { crop } from "Utils/resizer"
```

Comment:
@pepopowitz - can you make sure this can be run with JS disabled / is SSR friendly?

Reply:
(Actually, I think this will be fine since this component will only ever be client-side only! but this is more of an awareness thing that if something can be done in an SSR-friendly way its preferred over something that's client-side only.)

Reply:
For my understanding, can you elaborate a bit on what aspect of this worries you re SSR?

Reply:
I recall this having some kind of dependency on `react-sizeme` which caused a lot of trouble during the new publishing page roll out but looking at the util everything seems fine.

Reply:
Ok thanks 👍

Reply:
I think the issue was maybe coming from trying to make a responsive image that was 'perfectly' resized for the viewport and layout, which was requiring JS. I believe that was a previous iteration of artwork bricks/images at some point.

Now, we're just resizing images to some fixed dimensions that work for the desired layout and are reasonable (ie- cropped in this case to 185x80). For a more generic artwork grid they are resized to a max width of 3-400px, and similarly for a max height in a rail.